crate/src/lib.rs

```diff
@@ -15,10 +15,9 @@ mod page;
 use generated::css_classes::C;
 use guide::Guide;
 use page::partial::blender;
-use seed::{prelude::*, *};
+use seed::{browser::web_storage::LocalStorage, prelude::*, *};
```

Comment:
`LocalStorage` should be in `prelude`.
Also `cargo make verify` should work in `/crate` - maybe it finds similar issues.